pyomo/repn/plugins/nl_writer.py

```diff
@@ -107,16 +74,12 @@
 
 logger = logging.getLogger(__name__)
 
-# Feasibility tolerance for trivial (fixed) constraints
-TOL = 1e-8
+relocated_module_attribute('AMPLRepn', 'pyomo.repn.ampl.AMPLRepn', version='6.4.0.dev0')
```

Comment:
Was this set to an old version on purpose?
```suggestion
relocated_module_attribute('AMPLRepn', 'pyomo.repn.ampl.AMPLRepn', version='6.7.4.dev0')
```